API_design.md

```diff
@@ -0,0 +1,256 @@
+ # Restful API
+
+## Header
+
+### login_header
+```
+Host: api.orange.com
+Authorization: Bearer ACCESS_TOKEN
+Content-Type: application/json
+Accept: application/json
+User-Agent: OrangeApp/1.0
```

Comment:
Most of time, we don't specific the `User-Agent`.

Reply:
Oh oh i got it